styles.css

```diff
@@ -1,26 +1,221 @@
+*{
+    font-family: Arial, Helvetica, sans-serif;
+}
 body{
     margin: 0;
     padding: 0;
+    height: 100%;
+    width: 100%;
+}
+p{
+    margin: 0;
+    margin-left: 16px;
 }
 
-.nav{
+hr{
+    color: #7f8c8d;
+    background-color: #7f8c8d;
+    margin-bottom: 16px;
+    margin-top: 16px;
+}
 
+section{
+    float:left;
+    display:block;
+}
+aside{
+    float:right;
+    width:33%;
+    display: block;
 }
 
+
+/* Suggested Video Section */
 .side-videos{
-    border: solid 1px red;
-    margin: 8px;
+    margin: 1px;
+    margin-right: 3%;
+    
 }
 .inner-side-videos{
-    padding-top: 24px;
-    padding-right: 24px;
+    padding-top: 1px;
+    padding-right: 1%;
+    margin-right: 1px;
+}
+p,.up-next-video{
```

Comment:
Ok, we got a problem here. The thing is, on line 10 you also define style properties for `p`. The result, is that the rules from this block 'overwrite' the rules from the block on line 10 and that is hard to spot and debug.